src/ptree_readers/read_property.cxx

```diff
@@ -25,6 +25,6 @@ tablator::Property tablator::ptree_readers::read_property(
         }
         ++child;
     }
-
+    value.assign(prop_tree.get_value<std::string>());
```

Comment:
I don't understand the addition of this line. Doesn't this assignment overwrite the value assigned by the loop over property tree children?

Reply:
`prop_tree` shouldn't have children that aren't attributes.  I'll simplify the function.  Thanks.

Reply:
`prop_tree` shouldn't have children that aren't attributes.  I'll simplify the function.  Thanks.